src/components/Comments/AddComment.jsx

```diff
@@ -0,0 +1,60 @@
+import { useContext, useState } from "react";
+import { UserContext } from "../context/Username";
+import { postComment } from "../utils/api";
+
+const AddComment = props => {
+  const { article_id, setComments } = props;
+  const { username } = useContext(UserContext)
+  const [isError, setIsError] = useState(false);
+  const [isTooLong, setIsTooLong] = useState(false)
+  const [isSuccessful, setIsSuccessful] = useState(false)
+  const [comment, setComment] = useState('')
+  const maxChar = 300;
+  let message = <p></p>
+
+  const handleSubmit = e => {
+    e.preventDefault()
+    const toSend = {username, body: comment}
+    postComment(article_id, toSend)
+      .then(({comment}) => { 
+        setIsSuccessful(true)
+        setComment('')
+        setIsError(false)
+        setComments(c => {
+          return [comment, ...c]
+        })
+      })
+      .catch(() => {
+        setIsError(true)
+      })
```

Comment:
Currently I'm able to mash the add comment button and post loads of empty comments, can you make it so users are prevented from entering empty comments.